sbi/inference/__init__.py

```diff
@@ -13,12 +13,8 @@
 
 from sbi.inference.abc.mcabc import MCABC
 from sbi.inference.abc.smcabc import SMCABC
-from sbi.inference.base import (  # noqa: F401
-    NeuralInference,
-    check_if_proposal_has_default_x,
-    infer,
-    simulate_for_sbi,
-)
+from sbi.inference.base import NeuralInference  # noqa: F401
+from sbi.inference.base import check_if_proposal_has_default_x, infer, simulate_for_sbi
```

Comment:
I am a bit surprised about the black changes.

Reply:
In particular, I do not get these changes when I run `black` (v24.2.0) locally.

Reply:
I am surprised as well, but it seems to be due to `isort` and because of the comment.